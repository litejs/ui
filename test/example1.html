<!DOCTYPE html>
<html><head><title>App</title></head><body><script type="ui">

h1 LiteJS <is> simple!
p <!-- commented-out! -->

</script>
<script src="../load.js"></script>
<script src="../polyfill/index.js"></script>
<script src="../index.js"></script>
</body></html>
