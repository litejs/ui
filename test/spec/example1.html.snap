<body>
<script src="../load.js"></script>
<script src="../polyfill/index.js"></script>
<script src="../index.js"></script>
<h1 data-bind="txt:_('LiteJS &lt;is> simple!',data)">LiteJS &lt;is> simple!</h1><p data-bind="txt:_('&lt;\!-- commented-out! -->',data)">&lt;!-- commented-out! --></p></body>